src/client/client4.ts

```diff
@@ -667,58 +733,58 @@ export default class Client4 {
         } else {
             queryStringObj = {in_channel: channelId, page, per_page: perPage};
         }
-        return this.doFetch(
+        return this.doFetch<UserProfile[]>(
             `${this.getUsersRoute()}${buildQueryString(queryStringObj)}`,
             {method: 'get'},
         );
     };
 
-    getProfilesInGroupChannels = async (channelsIds: string[]) => {
+    getProfilesInGroupChannels = (channelsIds: string[]) => {
         this.trackEvent('api', 'api_profiles_get_in_group_channels', {channelsIds});
 
-        return this.doFetch(
+        return this.doFetch<{[channelId: string]: UserProfile[]}>(
```

Comment:
You can shorten this to `Record<string, UserProfile[]>`, it's basically the TypeScript helper for a dictionary.